common/locales/en/pets.json

```diff
@@ -1,10 +1,14 @@
 {
     "pets": "Pets",
+    "activePet": "Active Pet",
+    "noActivePet": "No Active Pet",
```

Comment:
Will do once not on mobile
On Mon, Sep 12, 2016 at 9:32 PM Blade Barringer <notifications@github.com>
wrote:

> In common/locales/en/pets.json
> <https://github.com/HabitRPG/habitrpg/pull/7768#discussion_r78488069>:
>
> > @@ -1,10 +1,14 @@
> >  {
> >      "pets": "Pets",
> > +    "activePet": "Active Pet",
> > +    "noActivePet": "No Active Pet",
>
> Let's remove the strings that aren't used then.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7768/files/72a6d73a9be23fd72ba98fbe74b2f14fbc840bfb#r78488069>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALz_8zh6M6g5Dbq85cPkUP-NbDz8bMbgks5qpgtQgaJpZM4JIjd_>
> .
>